### ISSUE TITLE
More prominent verified badge and better explanation of what it means

### ISSUE THREAD

**Issue by therealjuan** [canonical-websites/snapcraft.io#868]
The badge used to identify a verified publisher is quite small and the hover message could explain a little bit further what it means from a snapcraft point of view.

**Comment by solazio** [comment]
@Lukewh is he refering to the current badge or it was and older badge we had? 

![image.png](https://images.zenhubusercontent.com/5cb4608f97df763a9090bdec/c4926d42-f13a-422d-9216-37ba81138a0e)

**Comment by Lukewh** [comment]
> @Lukewh is he refering to the current badge or it was and older badge we had?

Yeah. Maybe we could get @danielmutis to have a look before you start on it?
Not sure a tooltip with more details is the best solution

**Comment by jkfran** [comment]
We are going to work on this for this cycle. It will be tracked in a different issue.
